sdk/python/kfp/components/component_decorator_test.py

```diff
@@ -75,3 +78,21 @@ def test_packages_to_install_with_custom_index_url(self):
         self.assertTrue('numpy' in concat_command and
                         'tensorflow' in concat_command)
         self.assertTrue('https://pypi.org/simple' in concat_command)
+
+    def test_output_component_file_parameter(self):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            filepath = os.path.join(tmpdir, 'my_component.yaml')
+            with self.assertWarnsRegex(
+                    DeprecationWarning,
+                    r'output_component_file parameter is deprecated and will eventually be removed\.'
+            ):
+                comp = component(func, output_component_file=filepath)
```

Comment:
Yes, we probably ought to. I took this approach to avoid having to redefine the decorated function in each test case, but the boilerplate is probably worth it.